targets/pybadge.json

```diff
@@ -0,0 +1,8 @@
+{
+    "inherits": ["atsamd51j19a"],
+    "build-tags": ["sam", "atsamd51j19a", "pybadge"],
```

Comment:
`"sam", "atsamd51j19a"` are unnecessary, they are already part of the inherited target.

Reply:
If I remove them, I get : 

```
error: no buildable Go source files in /workspace/go/src/github.com/tinygo-org/tinygo/src/device/sam
```

And I saw them in every other target, for example https://github.com/tinygo-org/tinygo/blob/master/targets/feather-m4.json#L3

Reply:
Ah, interesting. Seems to be a bug on this line:
https://github.com/tinygo-org/tinygo/blob/a44d74d4c05b771d1c013330949a5689855492aa/targets/atsamd51j19a.json#L4

It should have `atsamd51j19a` added to the build tags.

In that case, that can be fixed in a separate PR.

Reply:
Adding "atsamd54j19a" to atsamd51j19a.json works, but I think other targets have missing tags too, for example: https://github.com/tinygo-org/tinygo/blob/master/targets/atsamd21g18a.json

After that, I guess some tags could be removed from others targets too: https://github.com/tinygo-org/tinygo/blob/master/targets/feather-m4.json